.github/workflows/create-block.yml

```diff
@@ -33,5 +33,5 @@ jobs:
 
             - name: npm install, build, format and lint
               run: |
-                  npm ci
+                  npm install
```

Comment:
Why are we changing to `npm install` here (and in other workflows)? It may attempt to resolve/update dependencies and lead to unpredictable results while `npm ci` only reads from the `package-lock.json`.

Reply:
These are just temporary changes, sorry for the confusion. They were part of ab60daa1bcf439b41c817476920e48240cee561a and their purpose is to create a new npm cache. 
The changes will then be reverted before submitting the PR for review.

Reply:
They're to workaround a separate issue where an `npm ERR! cb() never called!` error is triggered.